packages/mobile/src/account/Settings.tsx

```diff
@@ -0,0 +1,344 @@
+  lockWithPinEnabled: boolean
+  fornoEnabled: boolean
+  gethStartedThisSession: boolean
+  preferredCurrencyCode: LocalCurrencyCode
+}
+
+type OwnProps = StackScreenProps<StackParamList, Screens.Settings>
+
+type Props = StateProps & DispatchProps & WithTranslation & OwnProps
+
+const mapStateToProps = (state: RootState): StateProps => {
+  return {
+    backupCompleted: state.account.backupCompleted,
+    account: state.web3.account,
+    devModeActive: state.account.devModeActive || false,
+    e164PhoneNumber: state.account.e164PhoneNumber,
+    analyticsEnabled: state.app.analyticsEnabled,
+    numberVerified: state.app.numberVerified,
+    pincodeType: pincodeTypeSelector(state),
+    lockWithPinEnabled: state.app.lockWithPinEnabled,
+    fornoEnabled: state.web3.fornoMode,
+    gethStartedThisSession: state.geth.gethStartedThisSession,
+    preferredCurrencyCode: getLocalCurrencyCode(state),
+  }
+}
+
+const mapDispatchToProps = {
+  revokeVerification,
+  setNumberVerified,
+  resetAppOpenedState,
+  setAnalyticsEnabled,
+  toggleBackupState,
+  devModeTriggerClicked,
+  setLockWithPinEnabled,
+  toggleFornoMode,
+}
+
+interface State {
+  fornoSwitchOffWarning: boolean
+  promptFornoModal: boolean
+}
+
+export class Account extends React.Component<Props, State> {
+  componentDidMount() {
+    const promptFornoModal = this.props.route.params?.promptFornoModal
+    if (promptFornoModal) {
+      this.setState({
+        promptFornoModal,
+      })
+    }
+  }
+
+  goToProfile = () => {
+    CeloAnalytics.track(CustomEventNames.edit_profile)
+    this.props.navigation.navigate(Screens.Profile)
+  }
+
+  goToLanguageSetting = () => {
+    this.props.navigation.navigate(Screens.Language, { fromSettings: true })
+  }
+
+  goToLocalCurrencySetting = () => {
+    this.props.navigation.navigate(Screens.SelectLocalCurrency)
+  }
+
+  goToLicenses = () => {
+    this.props.navigation.navigate(Screens.Licenses)
+  }
+
+  goToSupport = () => {
+    this.props.navigation.navigate(Screens.Support)
+  }
+
+  resetAppOpenedState = () => {
+    this.props.resetAppOpenedState()
+    Logger.showMessage('App onboarding state reset.')
+  }
+
+  toggleNumberVerified = () => {
+    this.props.setNumberVerified(!this.props.numberVerified)
+  }
+
+  revokeNumberVerification = async () => {
+    if (this.props.e164PhoneNumber && !isE164Number(this.props.e164PhoneNumber)) {
+      Logger.showMessage('Cannot revoke verificaton: number invalid')
+      return
+    }
+    Logger.showMessage(`Revoking verification`)
+    this.props.revokeVerification()
+  }
+
+  toggleBackupState = () => {
+    this.props.toggleBackupState()
+  }
+
+  showDebugScreen = () => {
+    this.props.navigation.navigate(Screens.Debug)
+  }
+
+  getDevSettingsComp() {
+    const { devModeActive } = this.props
+
+    if (!devModeActive) {
+      return null
+    } else {
+      return (
+        <View style={styles.devSettings}>
+          {/* <View style={style.devSettingsItem}>
```

Comment:
remove? or add a comment

Reply:
I'm not sure why it was commented in the first place? I just moved `dev` related things untouched.

Reply:
It's commented because it broke a while back but this is something we'd like to re-enable

Reply:
Cool, that would be awesome for testing from a clean unverified state 👍